### PR TITLE
gameplay scene

### PR DESCRIPTION
**Pull request by deathfated**
(pull request yg satu lagi abaikan)
gameplay scene init. ganti namespace script di gameplay & boot. scene baru. scene lama diganti namanya pake (ClickGame)

### CODE REVIEW

File: Assets/Scripts/Gameplay/GameplayLauncher.cs [hunk 1]
```diff
@@ -1,47 +1,46 @@
...
+              };
         }
 
-        protected override IEnumerator InitSceneObject()
+        protected override IController[] GetSceneDependencies()
         {
-            _clickGame.SetView(_view.ClickGameView);
-            _soundFx.SetView(_view.SoundFxView);
-            yield return null;
+            throw new System.NotImplementedException();
```
Comment (Leguna) [inline Assets/Scripts/Gameplay/GameplayLauncher.cs]: Bahaya gengs

File: Assets/Scripts/Gameplay/GameplayLauncher.cs [hunk 2]
```diff
@@ -1,47 +1,46 @@
...
 
-
-        protected override IConnector[] GetSceneConnectors()
+        protected override IEnumerator InitSceneObject()
         {
-            return new IConnector[]
-            {
-                new GameplayConnector()
-            };
+            throw new System.NotImplementedException();
```
Comment (Leguna) [inline Assets/Scripts/Gameplay/GameplayLauncher.cs]: Ini Jugah

File: Assets/Scripts/Gameplay/GameplayConnector.cs [hunk 3]
```diff
@@ -1,29 +1,22 @@
...
-            _saveData.OnUpdateCoin(message.Coin);
-            _soundFx.OnUpdateCoin();
-        }
-
+        //private SFXController _sfx;
+        
         protected override void Connect()
         {
-            Subscribe<UpdateCoinMessage>(OnUpdateCoin);
+            throw new System.NotImplementedException();
```
Comment (Leguna) [inline Assets/Scripts/Gameplay/GameplayConnector.cs]: di kosongin aj gpp atau kasih todo

File: Assets/Scripts/Gameplay/GameplayConnector.cs [hunk 4]
```diff
@@ -1,29 +1,22 @@
...
         }
 
         protected override void Disconnect()
         {
-            Unsubscribe<UpdateCoinMessage>(OnUpdateCoin);
+            throw new System.NotImplementedException();
```
Comment (Leguna) [inline Assets/Scripts/Gameplay/GameplayConnector.cs]: ini jugah

**Review by Leguna — CHANGES_REQUESTED** [review]
Fix Comment baru boleh merge

**Comment by Leguna** [comment]
@deathfated jangan lupa di assign to yourself PRnyah

**Review by deathfated — COMMENTED** [review]
sip sip

**Review by Leguna — APPROVED** [review]
Looks good to me.

**Pull request merged**